spec/api-net-spec.ts

```diff
@@ -1,11 +1,15 @@
 import { expect } from 'chai';
+import * as dns from 'dns';
 import { net, session, ClientRequest, BrowserWindow, ClientRequestConstructorOptions } from 'electron/main';
 import * as http from 'http';
 import * as url from 'url';
 import { AddressInfo, Socket } from 'net';
 import { emittedOnce } from './events-helpers';
 import { defer, delay } from './spec-helpers';
 
+// See https://github.com/nodejs/node/issues/40702.
+dns.setDefaultResultOrder('ipv4first');
```

Comment:
This "fix" should be reverted. It causes problems for lots of people using IPv6.

Reply:
Better use `net.autoSelectFamily` instead which enabled Happy Eyeballs according to RFC 8305.

Reply:
Also why is this change not addressed in the release notes in this PR? 

This change is not in the upstream and some downstream projects where expecting this option to change as per [v17 release notes](https://nodejs.org/es/blog/release/v17.0.0/#other-notable-changes). 

If there is a problem with it, it should raised as an issue of this project, discussed with the community and fixed accordingly.  Not by silently hiding it in large change.

Reply:
Fully agree. This has to be reverted.